src/pages/index.js

```diff
@@ -15,9 +17,18 @@ export default function Home() {
         <meta name="description" content="Generated by create next app" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
+
+      </Head> 
+      
+      <main>
+        <CustomizableAside />
+      </main> 
+      
+
       </Head>
       <main className={styles.main}></main>
       <FooterBar />
```

Comment:
You call `FooterBar` here but it is not imported above.

Reply:
I will remove it